src/Libraries/Microsoft.Extensions.Http.Resilience/Polly/HttpRetryStrategyOptions.cs

```diff
@@ -21,8 +22,10 @@ public class HttpRetryStrategyOptions : RetryStrategyOptions<HttpResponseMessage
     /// Initializes a new instance of the <see cref="HttpRetryStrategyOptions"/> class.
     /// </summary>
     /// <remarks>
-    /// By default, the options are set to handle only transient failures,
-    /// that is, timeouts, 5xx responses, and <see cref="HttpRequestException"/> exceptions.
+    /// By default, the options are configured to handle only transient failures.
+    /// Specifically, this includes HTTP status codes 408, 429, 500 and above, 
+    /// as well as <see cref="HttpRequestException"/> and <see cref="TimeoutRejectedException"/> exceptions.
+    /// Additionally, if the response includes a <c>Retry-After</c> header, its value will be used to determine the delay before retrying.
```

Comment:
Do we really want to include this information here? It is already included as part of xml-doc of `ShouldRetryAfterHeader` property.

Reply:
I removed this part.

Reply:
I wanted to highlight the fact that ShouldRetryAfterHeader is true through this constructor. But as long as there is only one constructor, this part is not necessary. I removed it.